orttraining/orttraining/test/python/orttraining_test_backend_api.py

```diff
@@ -0,0 +1,161 @@
+import os
```

Comment:
are these tests being triggered to run in the CI?

Reply:
No. Talked to Baiju and we agreed to use local test for this PR